authors/apps/core/token.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from datetime import timedelta
+import jwt
+from authors.settings import SECRET_KEY
+def encode_auth_token(user_id):
+    """
+    Generates the Auth Token
+    :return: string
+    """
+
+    try:
+        payload = {
+            'exp':
+            datetime.datetime.utcnow() + datetime.timedelta(minutes=60),
```

Comment:
I would advice having the time delta as a variable in settings.py.

Reply:
oh yeah thanks for the feedback 👍 😄

Reply:
Nice Progress YAyha......Just make change gitau has recommended.